docs/source/reference/cloud-permissions.rst

```diff
@@ -100,30 +107,36 @@ AWS accounts can be attached with a policy that limits the permissions of the ac
             }
         ]
     }
-5. Click “Next: Tags” and follow the instructions to create the policy.
-6. Go back to the previous window and click on the refresh button, and you can now search for the policy you just created.
+
+4. Click **Next: Tags** and follow the instructions to finish creating the policy. You can give the policy a descriptive name, such as ``minimal-skypilot-policy``.
+5. Go back to the previous window and click on the refresh button, and you can now search for the policy you just created.
 
 .. image:: ../images/screenshots/aws/aws-add-policy.png
     :width: 80%
     :align: center
     :alt: AWS Add Policy
-7. [Optional] If you would like to have your users access the s3 bucket. You can additionally attach the S3 access, such as the "AmazonS3FullAccess" policy.
+
+6. [Optional] If you would like to have your users access the s3 bucket. You can additionally attach the S3 access, such as the "AmazonS3FullAccess" policy.
 
 .. image:: ../images/screenshots/aws/aws-s3-policy.png
     :width: 80%
     :align: center
     :alt: AWS Add S3 Policy
 
-8. Click on "Next" and follow the instructions to create the user.
+7. Click on **Next** and follow the instructions to create the user.
 
 With the steps above you are almost ready to have the users in your organization to use SkyPilot with the minimal permissions.
 
 **One more thing** to do is to create a single service account "skypilot-v1" for all the users in your organization. There are two ways to accomplish this:
```

Comment:
Should we replace "service account" with "IAM role" in this doc? This is the AWS terminology (i.e., I found the role in https://us-east-1.console.aws.amazon.com/iamv2/home?region=us-east-1#/roles).

Reply:
Good point! Changed.